cmd/seccompagent/seccompagent.go

```diff
@@ -41,12 +44,14 @@ var (
 	socketFile    string
 	resolverParam string
 	logflags      string
+	monitorListen string
 )
 
 func init() {
 	flag.StringVar(&socketFile, "socketfile", "/run/seccomp-agent.socket", "Socket file")
 	flag.StringVar(&resolverParam, "resolver", "", "Container resolver to use [none, demo-basic, kubernetes]")
 	flag.StringVar(&logflags, "log", "info", "log level [trace,debug,info,warn,error,fatal,color,nocolor,json]")
+	flag.StringVar(&monitorListen, "monitor-listen", "", "[host]:port to listen on for monitoring, empty means no monitoring port")
```

Comment:
I see that many projects use `--metrics-bind-address`.

For example, [kube-proxy](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/). Could you use the same flag name?

```diff
-	flag.StringVar(&monitorListen, "monitor-listen", "", "[host]:port to listen on for monitoring, empty means no monitoring port")
+	flag.StringVar(&metricsBindAddr, "metrics-bind-address", "", "The IP address with port for the metrics server to serve on (set to '0.0.0.0:10249' for all IPv4 interfaces and '[::]:10249' for all IPv6 interfaces). Set empty to disable.")
```